web/src/message_lists.ts

```diff
@@ -130,7 +131,38 @@ export function save_pre_narrow_offset_for_reload(): void {
     }
 }
 
+export function calculate_timestamp_widths(): void {
+    const $temp_time_div = $("<div>");
+    $temp_time_div.attr("id", "calculated-timestamp-widths");
+    // Size the div to the width of the largest timestamp
+    $temp_time_div.css({width: "max-content", visibility: "hidden"});
```

Comment:
This is adding extra height to the page and causing scrollbar bugs ([discussion](https://chat.zulip.org/#narrow/stream/9-issues/topic/Recent.20conversations.20shifted.20up.20on.20navigating.20from.20narrow/near/1773606.2E01)).